src/pipeline.ml

```diff
@@ -74,6 +74,9 @@ let install_compiler_df ~distro ~arch ~switch ?windows_port opam_image =
     let additional_packages = Ocaml_version.Opam.V2.additional_packages switch in
     String.concat "," (Printf.sprintf "%s.%s" package_name package_version :: additional_packages)
   in
+  (match os_family with
+   | `Linux -> parser_directive (`Syntax "docker/dockerfile:1.4")
```

Comment:
Does it mean that we don't support the other syntax anymore? Maybe it's a dummy question but will the Dockerfile be able to be executed with an older docker?

Reply:
The new syntax is an extension, the "old" syntax is still there. This new syntax is opt-in though, and indeed it does require a Docker with BuildKit supported (at least 18.09, released on 2018-11). Support should be ok.